### ISSUE TITLE
Fails when two tables have the same name in different schemas

### ISSUE THREAD

**Issue by Bathlamos** [andrewfulrich/postgerd#1]
I admit that this might not be a very common situation, but thought I'd report nonetheless.

If I create the following two tables in two different schemas (I don't think that the foreign key is necessary to trigger this error, but including nonetheless):

```
create table a (
    id UUID primary key default gen_random_uuid()
);

create table b (
    id UUID primary key default gen_random_uuid(),
    a_id UUID not null references a(id)
);
```

Then I encounter the following error

`ERROR:  more than one row returned by a subquery used as an expression`

**Comment by andrewfulrich** [comment]
Good catch! I'll take a look. Thanks for posting!

**Comment by andrewfulrich** [comment]
I wasn't able to reproduce the error that you got, but that could be due to a code change that occurred since you posted. I was however able to see and fix a bug related to tables named the same with different schemas. Please try again and let me know if your issue is now resolved with v1.0.7

**Comment by andrewfulrich** [comment]
I'm going to go ahead and close this issue because I believe it's fixed with the latest code. My apologies again for not getting to the initial post sooner when it would have had a greater chance of response from the original author.
